functors task (1)/student/include/impl.h

```diff
@@ -16,16 +26,18 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand & brand)
```

Comment:
scalar types are usually passed by copy
also, if you don't modify params, use const

Reply:
fixed